material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -1523,27 +1546,42 @@ div.mermaid {
 }
 
 /* Edit this page styling */
-.actions {
+.md-content__button.md-icon {
   display: flex;
-  justify-content: center;
-  margin-top: 4em;
+  align-items: center;
+  gap: 8px;
+  font-size: 12px;
 }
 
-.md-typeset .actions a {
-  margin: 0;
-  font-size: 80%;
-  text-decoration: none;
-  display: flex;
-  align-items: center;
-  gap: 0.5em;
-  color: var(--md-default-fg-color);
-  opacity: 0.75;
+.md-content__button.md-icon svg {
+  fill: var(--heather);
+  width: 20px;
+  height: 20px;
```

Comment:
```suggestion
```

Noticed some weird things happening on wider screens. So let's just remove this, use the default size and switch the icon to material